src/main/kotlin/com/hibob/academy/dao/OwnerDao.kt

```diff
@@ -0,0 +1,36 @@
+package com.hibob.academy.dao
+
+import jakarta.inject.Inject
+import org.jooq.DSLContext
+import org.jooq.Record
+import org.jooq.RecordMapper
+import kotlin.random.Random
+
+class OwnerDao @Inject constructor(private val sql: DSLContext) {
+
+    private val owner = OwnerTable.instance
+    private val companyId = Random.nextLong()
+
+    private val ownerMapper = RecordMapper<Record, OwnerData> { record ->
+        OwnerData(
+            record[owner.name],
+            record[owner.companyId],
+            record[owner.employeeId]
+        )
+    }
```

Comment:
What about id?
Do we need to add id to the data class and the mapper?

Reply:
i dont think so, why do we need the id?